samples/transform_feedback_instanced.html

```diff
@@ -0,0 +1,380 @@
+            gl.deleteShader(fshaderTransform);
+
+            var varyings = ['v_offset', 'v_rotation'];
+            gl.transformFeedbackVaryings(programTransform, varyings, gl.SEPARATE_ATTRIBS);
+            gl.linkProgram(programTransform);
+
+            // check
+            var log = gl.getProgramInfoLog(programTransform);
+            if (log) {
+                console.log(log);
+            }
+
+            log = gl.getShaderInfoLog(vshaderTransform);
+            if (log) {
+                console.log(log);
+            }
+
+            // Setup program for draw shader
+            var programDraw = createProgram(gl, getShaderSource('vs-draw'), getShaderSource('fs-draw'));
+
+            var programs = [programTransform, programDraw];
+            return programs;
+        }
+
+        function transform() {
+            var programTransform = programs[PROGRAM_TRANSFORM];
+
+            // Toggle source and destination VBO
+            var sourceVBO = vertexBuffers[currentSourceIdx];
+            var sourceVAO = vertexArrays[currentSourceIdx];
+
+            var destinationVBO = vertexBuffers[(currentSourceIdx + 1) % 2];
+
+            gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, transformFeedback);
+
+            gl.useProgram(programTransform);
+
+            gl.bindVertexArray(sourceVAO);
+
+            // need to set to 0 when doing transform feedback
+            gl.vertexAttribDivisor(OFFSET_LOCATION, 0);
+            gl.vertexAttribDivisor(ROTATION_LOCATION, 0);
+
+            // Set transform feedback buffer
+            gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, OFFSET_LOCATION, destinationVBO[OFFSET_LOCATION]);
+            gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, ROTATION_LOCATION, destinationVBO[ROTATION_LOCATION]);
+
+            // Turn off rasterization - we are not drawing
+            gl.enable(gl.RASTERIZER_DISCARD);
+
+            // Update position and rotation using transform feedback
+            gl.beginTransformFeedback(gl.POINTS);
+            gl.drawArrays(gl.POINTS, 0, NUM_INSTANCES);
+            gl.endTransformFeedback();
+
+            // Restore state
+            gl.disable(gl.RASTERIZER_DISCARD);
+            gl.useProgram(null);
+            gl.bindBuffer(gl.ARRAY_BUFFER, null);
+            gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, null);
+            gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, null);
+
+            // Ping pong the buffers
+            currentSourceIdx = (currentSourceIdx + 1) % 2;
+        }
+
+        function render() {
+            // Rotate triangles
+            transform();
+
+            // Set the viewport
+            gl.viewport(0, 0, canvas.width, canvas.height - 10);
+
+            // Clear color buffer
+            gl.clearColor(0.0, 0.0, 0.0, 1.0);
+            gl.clear(gl.COLOR_BUFFER_BIT);
+
+            gl.bindVertexArray(vertexArrays[currentSourceIdx]);
+
+            // set back to 1 when draw instances

```

Comment:
Similar to above, it might be good to change the comment to "Attributes per-instance when drawing".